core/app/models/spree/order_updater.rb

```diff
@@ -73,7 +73,7 @@ def update_shipments
     end
 
     def update_payment_total
-      order.payment_total = payments.completed.includes(:refunds).map { |payment| payment.amount - payment.refunds.sum(:amount) }.sum
+      order.payment_total = payments.completed.sum(&:amount)
```

Comment:
I'm concerned that we're changing the meaning of this column. Both because existing applications may expect the amount to have the refunds subtracted, and because existing records in the database will already exist with the existing values.